rpcserver.go

```diff
@@ -1662,6 +1662,12 @@ func (r *rpcServer) NewAddr(ctx context.Context,
 			err)
 	}
 
+	// Sanity check early to ensure that the asset is known.
+	_, err = r.cfg.AddrBook.QueryAssetInfo(ctx, specifier)
```

Comment:
Should this be smarter and attempt to identify the asset automatically? No idea if that makes sense and out of scope of this PR.